src/engraving/dom/utils.cpp

```diff
@@ -169,7 +169,7 @@ Segment* Score::tick2segment(const Fraction& t, bool first, SegmentType st, bool
     }
 
     if (m == 0) {
-        LOGD("no measure for tick %d", tick.ticks());
+        LOGD() << "no measure for tick" << tick.ticks();
```

Comment:
Nitpick: space needed
```suggestion
        LOGD() << "no measure for tick " << tick.ticks();
```